hack/check/check-misspell.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
```suggestion
TF_REPO_PATH="$(git rev-parse --show-toplevel)"
```
lets simplify this and have a reference of repo root path and all the other paths can be concatenated from it